lib/index.ts

```diff
@@ -27,3 +27,10 @@ export const mainnetTokensMetadata = mainnet as TokenList
 export const testnetTokensMetadata = testnet as TokenList
 
 export default { mainnet: mainnetTokensMetadata, testnet: testnetTokensMetadata }
+
+export const ALPH: TokenInfo = {
+  id: '0',
```

Comment:
Perhaps we can use [this definition](https://github.com/alephium/alephium-web3/blob/master/packages/web3/src/constants.ts#L21) to keep the format of the token id consistent with other tokens?

Reply:
done!